resdk/analysis/differential_expressions.py

```diff
@@ -42,15 +42,14 @@ def cuffdiff(resource, annotation, genome=None, multi_read_correct=None, fdr=Non
         quartile
     :param str dispersion_method: options are: pooled, per-condition,
         blind, poisson
-    :param int threads: use this many processor threads
 
     """
     inputs = {'annotation': get_data_id(annotation)}
 
     input_objects = [annotation]
 
     if genome is not None:
-        inputs['genome'] = genome
+        inputs['genome'] = genome.id
```

Comment:
As already mentioned here https://github.com/genialis/resolwe-bio-py/pull/147#discussion_r179910866, this should work without `.id`. Please check why it doesn't.